Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1

```diff
@@ -43,6 +43,9 @@ Write-Verbose "configureTestAgentScriptLocation = $configureTestAgentScriptLocat
 $checkAgentInstallationScriptLocation = Join-Path -Path $currentDirectory -ChildPath "CheckTestAgentInstallation.ps1"
 Write-Verbose "checkAgentInstallationScriptLocation = $checkAgentInstallationScriptLocation"
 
+$downloadTestAgentScriptLocation = Join-Path -Path $currentDirectory -ChildPath "DownloadTestAgent.ps1"
+Write-Verbose "downloadTestAgentScriptLocation = $downloadTestAgentScriptLocation"
```

Comment:
what happens when new task works with old cmd let. we should validate and things should work